npm/iptm/iptm.go

```diff
@@ -78,6 +88,11 @@ func (iptMgr *IptablesManager) InitNpmChains() error {
 		metrics.SendErrorLogAndMetric(util.IptmID, "Error: failed to add AZURE-NPM chain to FORWARD chain. %s", err.Error())
 	}
 
+	// (TODO) need a review of all chains to make sure we are not adding duplicate rules.
+	// NPM should not add multiple TO/FROM rules along with DROPs in PORT chains,
```

Comment:
Still need these comments? Is it resolved in this PR?

Reply:
Yes we will not need this comment as we are taking care of these duplicate rules in this PR.